libraries/aws_iam_group.rb

```diff
@@ -21,12 +21,18 @@ def initialize(opts = {})
 
     catch_aws_errors do
       @resp = @aws.iam_client.get_group(group_name: opts[:group_name])
+      group_name = { group_name: opts[:group_name] }
+      iam_client = @aws.iam_client
 
       group       = @resp[:group]
       @group_name = group.group_name
       @group_id   = group.group_id
       @arn        = group.arn
       @users      = @resp[:users].map(&:user_name)
+      @inline_policy_names = iam_client.list_group_policies(group_name).policy_names
+    rescue Aws::IAM::Errors::NoSuchEntity
```

Comment:
`catch_aws_errors` catches this exception automatically, so this is not required.

Reply:
Tested and confirmed this line is not required. Thanks for the catch.